packages/server/modules/shared/repositories/roles.ts

```diff
@@ -28,6 +32,13 @@ export const getRolesFactory =
     return roles
   }
 
+export const getCachedRolesFactory = wrapFactoryWithCache({
```

Comment:
these will never change while the app is running

Reply:
(just have to make sure not to invoke this before the init role migrations are done...currently its fine, this is only used in authorizeResolver)